### PR TITLE
Fix bug with dragging on maps in Firefox

### LINKED ISSUE #5723: Competitions Map Bug (Firefox)
**Describe the bug**
When using the map option on the list of competitions to see past, present or future competitions, it's not possible to drag with the cursor to move around the map unless you do it pressing on a pin. It works perfectly on Safari (iOS) and Vivaldi (Ubuntu), I think the problem is on Firefox as I tried on Ubuntu and Windows having the same problem on both cases.

**To Reproduce (using Firefox)**
1. Go to 'Competitions' > 'All'
2. Click on the 'Map' button.
3. Try to move around pressing on a random part of the map. (Here is the problem)
4. Try to move around pressing on a pin and dragging.

**Expected behavior**
You should be able to move around pressing on any part of the map, as you can do on any other browser or device.

**Desktop:**
 - OS: Ubuntu 20.04.1 LTS and Windows 10.
 - Browser: Mozilla Firefox
 - Version: 80.0.1

**Other:**
I have tried on the example [here](https://leafletjs.com/) and it works fine.

### PR DESCRIPTION
**Pull request by louismeunier**
fixes #5723

### CODE REVIEW

**Comment by gregorbg** [comment]
It seems that you created your `firefox-map-drag-bug` branch on top of the other PR's rankings branch.
Please make sure this PR only contains the relevant commits by rebasing.

**Comment by gregorbg** [comment]
Also, judging by the test failures it seems that you accidentally introduced trailing whitespace in one SCSS file.

```
Running pre-commit hooks
Check for trailing whitespace....................[TrailingWhitespace] FAILED
Errors on modified lines:
/home/runner/work/worldcubeassociation.org/worldcubeassociation.org/WcaOnRails/app/assets/stylesheets/competitions.scss:229:  
```

**Comment by gregorbg** [comment]
Nevermind, just realised that the Bundler version isn't included in our README.
Install the appropriate version like so: `gem install bundler -v WHATEVER_VERSION`

File: WcaOnRails/app/assets/stylesheets/competitions.scss [hunk 1]
```diff
@@ -226,6 +226,10 @@ $venue-map-wrapper-height: 400px;
   &.competitions-map #competitions-admin {
     display: none;
   }
+  
+  #competitions-map {
```
Comment (jonatanklosko) [inline WcaOnRails/app/assets/stylesheets/competitions.scss:230]: There are other places where we use maps (like venue location maps under "Edit venues information" [in the schedule editor](https://staging.worldcubeassociation.org/competitions/WesternChampionship2021/schedule/edit)) and we may add more in the future.

It would be best to have a single global rule in `wca.scss` (where we currently keep such specific global styles) that would apply to all maps. Inspecting the rendered map it looks like there's always a root element with `leaflet-container` class, so if we target that with CSS, it will hopefully fix the issue globally.

**Review by jonatanklosko — COMMENTED** [review]
@louismeunier great find! The solution looks good, I'd just try to generalize it a bit :)

File: WcaOnRails/app/assets/stylesheets/competitions.scss (hunk 1 above)
Comment (louismeunier) [reply WcaOnRails/app/assets/stylesheets/competitions.scss]: Thanks for the feedback! Sorry for the messy commits, I'm not great at rebasing yet.

**Review by jonatanklosko — APPROVED** [review]
Awesome! :ship:
